Makefile

```diff
@@ -17,11 +17,11 @@ run:
 clean:
 	$(GOCLEAN)
 	rm -f $(BINARY_NAME)
-dep:
-    $(GOGET) github.com/aws/aws-sdk-go
-    $(GOGET) github.com/cloudflare/tableflip
-    $(GOGET) github.com/djherbis/atime
-    $(GOGET) github.com/gorilla/mux
-    $(GOGET) github.com/pkg/errors
-    $(GOGET) github.com/rcrowley/go-metrics
-    $(GOGET) github.com/spf13/viper
\ No newline at end of file
+dep: 
```

Comment:
This is a cosmetic change unrelated to the fsnotify feature. Please revert this file's changes.

Reply:
I had no possibility to run it according to such error: "Makefile:21: *** missing separator. Stop"
Also you have another PR that fixing it: https://github.com/kxlt/imageresizer/pull/19

Reply:
Indeed, I see the problem. It's good that you fixed it, but as a rule of thumb, please create a *separate commit* for stuff like this in your future contributions to **any project**.

It's a good practice not to include unrelated changes in your main commit, create separate commits for each individual change that makes sense. Especially do not make unnecessary whitespace changes alongside a feature commit.